helm/alfresco-identity-service/values.yaml

```diff
@@ -4,6 +4,8 @@ ingress:
 
 ingressHostName: "your ingress dns name"
 
+ALFRESCO_CLIENT_REDIRECT_URIS: "http://localhost*"
```

Comment:
Can we change this to:
```
clients:
    alfresco:
        redirectUris:
```

That's more inline with typical values file structure and will make it easier to add other properties or clients in the future.

This will obviously require a change in `realm-secret.yaml` to replace with `.Values.clients.alfresco.redirectUris`.